src/Spectre.libGaussianMixtureModelling/ExpectationMaximization.h

```diff
@@ -21,7 +21,7 @@ limitations under the License.
 #pragma once
 #include <random>
 #include "Spectre.libException/NullPointerException.h"
-#include "Spectre.libGaussianMixtureModelling/DataType.h"
+#include "Common/DataTypes.h"
```

Comment:
Please make sure it is included into our NuGet package. I still wonder whether we shouldn't just have such file inside `Spectre.libDataset` or such.